classes/rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
- [ ] the date instance variable should have a write attribute as well for this relationship to work correctly.